README.md

```diff
@@ -19,11 +19,23 @@ Install the Serilog (if not included already) and Serilog.Sinks.Raygun package i
 
 ------
 
-## Step 2 - Add logger configuration
+## Step 2 - Initialization
+There are two recommended options for initializing the Raygun Serilog Sink. Which option is best to use depends on the application. 
 
-Setup the logger configuration by passing in your API key. You can do this inside of Program.Main(), Global.asax, Application_Start, or how ever your application is set up with the main, start up running method of your application.
+### Option 1 - Using logger configuration
+
+You can initialize Raygun's Serilog Sink through a Logger Configuration. This should be done inside of the main entry point of your application - for instance, Program.Main(), Global.asax, Application_Start Etc. The exact entry point will differ between frameworks.
+**Minimum setup example:**
```

Comment:
If this is intended to be a kind of heading, we'll need a new line above it, otherwise it looks like:
![image](https://github.com/MindscapeHQ/serilog-sinks-raygun/assets/3094648/9cf4a696-a9fe-4c69-b600-2d340cfbca62)